training/distributed_training/pytorch/data_parallel/rnnt/RNNT_notebook.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.      aws ecr get-login-password --region us-east-1 | docker login \
</pre><p>Don't hard-code the region.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
fixed!